haystack/pipelines/base.py

```diff
@@ -528,11 +528,12 @@ def run(  # type: ignore
                         if queue.get(n):  # concatenate inputs if it's a join node
                             existing_input = queue[n]
                             if "inputs" not in existing_input.keys():
-                                updated_input: dict = {
-                                    "inputs": [existing_input, node_output],
-                                    "params": params,
-                                    "_debug": {**existing_input["_debug"], **node_output["_debug"]},
-                                }
+                                updated_input: dict = {"inputs": [existing_input, node_output], "params": params}
+                                if "_debug" in existing_input.keys() or "_debug" in node_output:
```

Comment:
`existing_input` and `node_output` are both dictionaries right? So I would suggest to check with the same approach whether they contain the `"_debug"` key. Or is there a specific reason to not do that?

Reply:
Oh yes, my bad...